big_tests/tests/connect_SUITE.erl

```diff
@@ -546,6 +568,45 @@ bind_server_generated_resource(Config) ->
     ?assert(is_binary(Resource)),
     ?assert(byte_size(Resource) > 0).
 
+same_resource_replaces_session(Config) ->
+    UserSpec = [{resource, <<"confict">>} | escalus_users:get_userspec(Config, alice)],
```

Comment:
`<<"confict">>` (no 'el' letter) vs `<<"conflict">>` a couple lines below.

Reply:
Ahh, but it's just a resource name, so it likely doesn't have to match with the error condition.

Reply:
Looks ugly nevertheless. :)